src/lib/browse/layers/CrossingsLayerControl.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+  import {
+    ColorLegend,
+    ExternalLink,
+    HelpButton,
+    InteractiveLayer,
+  } from "lib/common";
+  import { Checkbox } from "lib/govuk";
+  import { overwriteCircleLayer, overwritePmtilesSource } from "lib/maplibre";
+  import type { MapGeoJSONFeature } from "maplibre-gl";
+  import { map } from "stores";
+  import { colors } from "../colors";
+
+  const name = "crossings";
+  let show = false;
+
+  function tooltip(feature: MapGeoJSONFeature): string {
+    const crossingTypeString = feature.properties.crossing;
+    if (crossingTypeString === "no")
+      return "Location where crossing is impossible/illegal but where there is a clear desire line to cross.";
+    let result = "Crossing with the following features.";
```

Comment:
This looks out-of-date and almost never used. I think you can just  change all of this code to do `descriptions.get(crossingType) ?? crossingType`